testing/unit/test_faucetizer.py

```diff
@@ -58,7 +58,8 @@ def _update_port_config(self, behavioral_config, switch, port, vlan, role=None,
         port_config = behavioral_config['dps'][switch]['interfaces'][port]
         port_config['native_vlan'] = vlan
         port_config['acls_in'] = [f'role_{role}'] if role else []
-        port_config['acls_in'].append(tail_acl)
+        if tail_acl:
```

Comment:
is this functionally same as the line above?  append X if X is defined?   If so, can you use the same syntax in both places?

Reply:
I think the previous logic is wrong since role-based ACL is independent of tail ACL. Changed the logic a little bit.. PTAL..